Formula/magnetix.rb

```diff
@@ -7,6 +7,14 @@ class Magnetix < Formula
 
   depends_on :xcode => :build
 
+  # Port audio code from QTKit to AVFoundation
+  # Required since 10.12 SDK no longer includes QTKit.
+  # Submitted by email to the developer.
+  patch do
```

Comment:
@mistydemeo as you may have guessed, I'm just anticipating what @MikeMcQuaid will/would/may say here hehe